### PR TITLE
[Typing] Correct types in some modules

### PR DESCRIPTION
**Pull request by Diogo-Rossi**
# Description

Correct some types on the following modules:
* `MeshElement.py`
* `RegionAssembly.py`
* `MeshPart.py`
* `MdbBase.py`

Implements #2782 

# Backporting

This change should be backported to the previous releases.

- [x] 2016
- [x] 2017
- [x] 2018
- [x] 2019
- [x] 2020
- [x] 2021
- [x] 2022

# Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

**Review by haiiliin — APPROVED** [review]
Excellent!

File: src/abaqus/Region/RegionAssembly.py [hunk 1]
```diff
@@ -193,17 +192,17 @@ def Set(
...
+        xVertices: Optional[Sequence[ConstrainedSketchVertex]] = None,
         xEdges: Optional[Sequence[Edge]] = None,
         xFaces: Optional[Sequence[Face]] = None,
         referencePoints: Sequence[ReferencePoint] = (),
-        skinFaces: tuple = ...,
-        skinEdges: tuple = ...,
-        stringerEdges: tuple = ...,
+        skinFaces: Tuple[Tuple[str, Face], ...] = ...,
+        skinEdges: Tuple[Tuple[str, Edge], ...] = ...,
+        stringerEdges: Tuple[Tuple[str, Edge], ...] = ...,
```
Comment (haiiliin) [inline src/abaqus/Region/RegionAssembly.py:205]: Should this be something like `Tuple[Tuple[str, Sequence[Face]], ...]`?

Comment (Diogo-Rossi) [reply src/abaqus/Region/RegionAssembly.py:205]: Yes, you're right

Comment (Diogo-Rossi) [reply src/abaqus/Region/RegionAssembly.py]: Also changed in [2879e8d](https://github.com/haiiliin/abqpy/pull/2872/commits/2879e8d4fbcf5f414bb6ab0d011f997e418dd61d)

**Pull request merged**